packages/vscode-extension/package.nls.json

```diff
@@ -401,14 +401,14 @@
     "teamstoolkit.walkthroughs.steps.teamsToolkitBuildApp.title": "Build your first app",
     "teamstoolkit.walkthroughs.steps.teamsToolkitCreateFreeAccount.description": "To build app for Teams, you need a Microsoft account with custom app upload permissions. Don't have one? Create a Microsoft developer sandbox with the Microsoft 365 Developer Program.\n Notice that Microsoft 365 Developer Program requires Visual Studio subscriptions. [Learn more about Microsoft 365 Developer Program](https://learn.microsoft.com/en-us/office/developer-program/microsoft-365-developer-program)\n[Join Microsoft 365 Developer Program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)",
     "teamstoolkit.walkthroughs.steps.teamsToolkitCreateFreeAccount.title": "Create Microsoft 365 developer sandbox",
-    "teamstoolkit.walkthroughs.steps.teamsToolkitDeploy.description": "You can provision resources in the cloud and deploy your app's code to the provisioned resources. Finally, distribute your first app to Teams.\n[Open Lifecycle Commands](command:fx-extension.openLifecycleTreeview?%5B%22WalkThrough%22%5D)\n__Tip: Learn more about ['Lifecycle'](https://aka.ms/teamsfx-provision).__",
+    "teamstoolkit.walkthroughs.steps.teamsToolkitDeploy.description": "You can provision resources in the cloud and deploy your app's code to the provisioned resources. Finally, distribute your first app to Teams.\n[Open Lifecycle Commands](command:fx-extension.openLifecycleTreeview?%5B%22WalkThrough%22%5D)\n__Tip: Learn more about [Lifecycle](https://aka.ms/teamsfx-provision).__",
```

Comment:
Content is too long. 
Proposed change - Provision resources in the cloud, deploy your app's code to these resources, and then share your first app to Teams. \n[Open Lifecycle Commands](command:fx-extension.openLifecycleTreeview?%5B%22WalkThrough%22%5D)\n__Tip: Learn more about [Lifecycle](https://aka.ms/teamsfx-provision).__",
Logic: Imperative language urges users to take action rather than suggestive language.